### DISCUSSION TITLE
Fastest setup for toolchain - Docker Image?

### DISCUSSION THREAD

**Discussion by LeifBloomquist** [zeldin/OrangeCart#1] [Q&A] [answered]
Hi all, what's the quickest way to get up and running for development?

I ran across [OCDevEnvPi ](https://github.com/zeldin/OCDevEnvPi) for the Pi, but unfortunately I don't have one.  Could something like this be created, maybe for Docker for Linux or Windows?

Alternatively is there a master list of tools and applications required?

**Comment by jgrip** [comment, marked as answer]
I'm using https://github.com/YosysHQ/oss-cad-suite-build myself for my work. Has all the tools I need and more.
Easily installable on most OSes.
